packages/core/src/visitors.ts

```diff
@@ -201,13 +201,14 @@ type Oas2FlatVisitor = {
 const legacyTypesMap = {
   Root: 'DefinitionRoot',
   ServerVariablesMap: 'ServerVariableMap',
-  PathsMap: 'PathMap',
+  Paths: 'PathMap',
```

Comment:
Oooh, looks like we don't have a way to add one more legacy name: `PathsMap`..

Reply:
Yes, I couldn't do that. I'm not sure how important it is given we renamed it recently.

Reply:
Thank you.